### DISCUSSION TITLE
Getting the error Cannot destructure property 'store' of 'useReduxContext(...)' as it is null while deploying on vercel

### DISCUSSION THREAD

**Discussion by Ab-Qadir1998** [vercel/next.js#59697] [App Router]
I am using next js v14 and rtk query. Everything working fine in local dev environment but while deploying on vercel I got this. Tried almost everything in my knowledge but couldn't resolved the issue

**Comment by MrAndrushik** [comment]
I solved this problem by renaming the folder '/pages', apparently when using App Router in Next.js the 'pages' folder is also a reserved name
![image](https://github.com/vercel/next.js/assets/80775861/136ee462-805f-4fc6-a870-3e95fe29a047)

**Reply by BLazzeD21** [reply]
thx

**Reply by AnkurNayak** [reply]
Thanks you've saved my day!

**Reply by louischarles2000** [reply]
Thanks. This really helped!
